app/src/main/res/drawable/ic__arrow_back.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="512"
```

Comment:
I think the height here is the reason when I scroll up the toolbar doesn't collapse to normal height.